src/abstractdataframe/abstractdataframe.jl

```diff
@@ -422,17 +425,20 @@ describe(df)
 """
 StatsBase.describe(df::AbstractDataFrame; kwargs...) = describe(stdout, df; kwargs...)
 function StatsBase.describe(io::IO, df::AbstractDataFrame; stats::AbstractVector{Symbol} = 
-                            [:mean, :min, :median, :max, :nmissing, :eltype])
+                            [:mean, :min, :median, :max, :nunique, :nmissing, :eltype], 
+                            all = false)
```

Comment:
What about my suggestion to use an `:all` pseudo-stat instead?

Reply:
Oh sorry I didn't understand the first time. Will add.

Reply:
Have added it, and changed the test.